end-to-end-tests/specs/screenshot.spec.js

```diff
@@ -58,6 +58,13 @@ function runResultsTestSuite(prefix){
         assertScreenShotMatch(res);
     });
 
```

Comment:
lets also get one of the mRNA tab where you select something and bar chart shows up

Reply:
yeah I'll add that too, just trying to get this simple one to work consistently first